mlflow/metrics/genai/genai_metric.py

```diff
@@ -33,7 +33,7 @@ def _format_args_string(grading_context_columns: Optional[List[str]], eval_value
     return (
         ""
         if args_dict is None
-        else "\n".join(f"Provided {arg}: {arg_value}" for arg, arg_value in args_dict.items())
+        else "\n".join(f"key: {arg}\nvalue: {arg_value}" for arg, arg_value in args_dict.items())
```

Comment:
Nit, Is it possible that the arg_value might have newline in it? If so, might be useful to use two newline to separate.